scss/_tag.scss

```diff
@@ -0,0 +1,61 @@
+// Base class
+
+.tag {
+  display: inline-block;
+  padding: $tag-padding-y $tag-padding-x add($tag-padding-y, 1px); // Boosted mod
```

Comment:
`// Boosted mod` comments are meant to identify divergences from Bootstrap. Since you're creating a whole file and component, you shouldn't have any.